### ISSUE TITLE
Invariant Violation when working on multiple projects

### ISSUE THREAD

**Issue by olivergeorge** [dmotz/natal#16]
(I've been doing little "spike" projects to get the feel of different features and components.  There's a gotcha when you do this.  I'm noting here in case it helps someone else.)

I expect to be able to switch between two natal based projects by stopping the Simulator, switching to different directory and running "natal launch && natal repl" but this leads to an "Invariant Violation: Application XX has not been registered" when opening second project.

The solution seems to be to close the React Packager terminal window.

This might well not be a natal issue but perhaps it is possible to ensure projects don't overlap from React Packager's perspective.

(love your work)

**Comment by dmotz** [comment]
Yeah there's a brief mention in the readme that running multiple instances at once will cause issues with the packager.

The plan is to have Natal control the React packager directly so that when Natal quits, the packager it started will quit with it and you'll avoid the invariant violation you're seeing.

Stay tuned.
